MAVProxy/modules/mavproxy_arm.py

```diff
@@ -120,6 +120,25 @@ def cmd_disarm(self, args):
             0, # param6
             0) # param7
 
+    def all_checks_enabled(self):
+        ''' returns true if the UAV is skipping any arming checks'''
```

Comment:
On Wed, 7 Sep 2016, Francisco Ferreira wrote:

> > +    def all_checks_enabled(self):
> > +        ''' returns true if the UAV is skipping any arming checks'''
> 
> returns false

Darn!  I inverted that function from "any_checks_disabled()"!